src/main/java/com/example/springboot/BookStoreApplication.java

```diff
@@ -0,0 +1,11 @@
+package com.example.springboot;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class BookStoreApplication {
+    public static void main(String[] args) {
+        SpringApplication.run(BookStoreApplication.class, args);
```

Comment:
you should test your application: add, find, and print the first book to console